src/main/resources/hudson/security/GlobalMatrixAuthorizationStrategy/config.jelly

```diff
@@ -178,27 +178,29 @@ THE SOFTWARE.
       </tr>
     </table>
     <local:isEditable>
-      <input type="button" class="matrix-auth-add-button" value="${%Add user…}" id="${id}UserButton"
-            data-table-id="${id}"
-            data-type="USER"
-            data-message-prompt="${%userPrompt}"
-            data-message-empty="${%userEmpty}"
-            data-message-error="${%userError}"
-      />
-      <input type="button" class="matrix-auth-add-button" value="${%Add group…}" id="${id}GroupButton"
-            data-table-id="${id}"
-            data-type="GROUP"
-            data-message-prompt="${%groupPrompt}"
-            data-message-empty="${%groupEmpty}"
-            data-message-error="${%groupError}"
-      />
+      <div class="jenkins-!-margin-top-2">
```

Comment:
minor spacing fix, can extract if needed

Reply:
Is this CSS class one that I can rely on existing in the future?

Reply:
Yes it's a stable API, documented at:
https://weekly.ci.jenkins.io/design-library/Spacing/